### PR TITLE
[REVIEW][FEATURE][UWANG-165] - Card user information(detail)

### PR DESCRIPTION
**Pull request by fahmi-azzuhri**
[REVIEW][FEATURE][UWANG-165] - Card user information(detail)

#### Description

> Description of features, bugs, improvements or any changes in this MR.

#### Ticket Reference(s)

[UWANG-165](https://bluehabit.atlassian.net/browse/UWANG-165)

##### UI Changes

> Screenshot or video of UI-related changes
localhost-3000

##### Coverage

> Test coverage screenshot.

N/A

#### Note(s)

> Mention other user or extra note unrelated to the MR here.

[UWANG-165]: https://bluehabit.atlassian.net/browse/UWANG-165?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[UWANG-165]: https://bluehabit.atlassian.net/browse/UWANG-165?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: components/UserInformation.vue [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
```
Comment (triandamai) [inline components/UserInformation.vue]: gausah pake `defineComponent` pake syntax sugar aja `<script setup lang="ts">` @fahmi-azzuhri

File: components/UserInformation.vue [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    data() {
+        return {
+            birthDate: new Date('1990-01-01'),
```
Comment (triandamai) [inline components/UserInformation.vue]: pake `ref` @fahmi-azzuhri

File: components/UserInformation.vue [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            registerDate: new Date('2023-02-23'),
+            followerCount: 1000,
+            postCount: 500,
+            country: 'Indonesia',
+            lastActive: '2024-02-23',
+        };
+    },
+    computed: {
+        userName(): string {
```
Comment (triandamai) [inline components/UserInformation.vue]: ga perlu computed ya kalo udah pake ref

**Review by triandamai — CHANGES_REQUESTED** [review]
benerin dulu @fahmi-azzuhri

**Comment by fahmi-azzuhri** [comment]
done bang @triandamai

**Review by triandamai — APPROVED** [review]

**Pull request merged**